.github/workflows/sync_develop_master.yml

```diff
@@ -0,0 +1,75 @@
+name: Branch synchronizer
+
+on:
+  schedule:
+    - cron:  '0 04 * * *'
+
+jobs:
+  sync-master-with-released:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout released
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+          fetch-depth: 0
+          ref: released
+      - name: Check for diff
+        id: check-diff
+        run: |
+          git fetch origin master
+          git checkout master
+          if [ -n "$(git diff master...released)" ]; then
+            echo "Master needs sync with released!";
+            echo "create_pull_request=1" >> $GITHUB_OUTPUT
+          else
+            echo "Master is already up to date.";
+            echo "create_pull_request=0" >> $GITHUB_OUTPUT
+          fi
+      - name: Create Pull Request if necessary
+        if: ${{ steps.check-diff.outputs.create_pull_request == 1 }}
+        uses: repo-sync/pull-request@v2.6.2
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          source_branch: released
+          destination_branch: master
+          pr_title: Merge released into master
+          pr_body: Synchronizes changes of released branch into master branch.
+          pr_reviewer: Maintainers
+          pr_label: "synchronization"
```

Comment:
Since the workflows don't appear to run immediately, I think we can avoid setting the label (that way we don't need to remove it, and re-set it)